jsx/src/util/withAPI.js

```diff
@@ -19,6 +19,8 @@ const withAPI = withProps(() => ({
     names.map((e) => jhapiRequest("/users/" + e + "/server", "DELETE")),
   addToGroup: (users, groupname) =>
     jhapiRequest("/groups/" + groupname + "/users", "POST", { users }),
+  updateProp: (propobject, groupname) =>
```

Comment:
Update properties looks great. We'd also need a `GET` properties method so the user can see what's already stored. Right now it comes back blank when a user revisits the edit page.

Reply:
That is weird, for me the properties are displayed when leaving and entering the edit page again. I obtain the properties directly from the group_data state, because it also contains group_data.properties. I can also create a direct GET method for the properties only as well. 
If I recall correctly, this should happen here: [https://github.com/vladfreeze/jupyterhub/blob/group_property_feature/jsx/src/components/GroupEdit/GroupEdit.jsx#L53-L54](url)